conv.go

```diff
@@ -217,6 +217,11 @@ func expandBriefTargetDuration(period string) ([]string, error) {
 //   - error: An error if any step of the conversion process fails.
 func (conv *Conv) ConvertDuration() (string, error) {
 	var err error
+	isNegativeDuration := false
+	if conv.Source[0] == '-' {
+		conv.Source = conv.Source[1:]
+		isNegativeDuration = true
+	}
```

Comment:
This is not a good way, it could panic

I strongly suggest you this

```suggestion
	if s, found := strings.CutPrefix(conv.Source, "-"); found {
		conv.Source = s
		isNegativeDuration = true
	}
```

But then, you could write this, but it's a bit more cryptic

```suggestion
	conv.Source, isNegativeDuration := strings.CutPrefix(conv.Source, "-")
```

Reply:
Great suggestion.

I implemented it here:

https://github.com/jftuga/DateTimeMate/pull/19